src/Symfony/Component/PropertyAccess/PropertyAccess.php

```diff
@@ -19,39 +21,30 @@
 final class PropertyAccess
 {
     /**
-     * Creates a property accessor with the default configuration.
+     * Creates a property accessor.
+     *
+     * For dealing with several property accessor configured differently, use
+     * the createPropertyAccessorBuilder() method instead.
      *
-     * @param bool $throwExceptionOnInvalidIndex
-     * @param bool $magicCall
+     * @param bool                        $throwExceptionOnInvalidIndex
+     * @param bool                        $magicCall
+     * @param CacheItemPoolInterface|null $cacheItemPool
      *
      * @return PropertyAccessor The new property accessor
      */
-    public static function createPropertyAccessor($throwExceptionOnInvalidIndex = false, $magicCall = false)
+    public static function createPropertyAccessor($magicCall = false, $throwExceptionOnInvalidIndex = false, CacheItemPoolInterface $cacheItemPool = null)
     {
-        return self::createPropertyAccessorBuilder($throwExceptionOnInvalidIndex, $magicCall)->getPropertyAccessor();
+        return new PropertyAccessor($magicCall, $throwExceptionOnInvalidIndex, $cacheItemPool);
     }
 
     /**
      * Creates a property accessor builder.
      *
-     * @param bool $enableExceptionOnInvalidIndex
-     * @param bool $enableMagicCall
-     *
      * @return PropertyAccessorBuilder The new property accessor builder
      */
-    public static function createPropertyAccessorBuilder($enableExceptionOnInvalidIndex = false, $enableMagicCall = false)
+    public static function createPropertyAccessorBuilder()
```

Comment:
These args have been introduced in #18977 that has been merged the last week, the changes are present only in the master branch (3.2-dev). Is it even considered as a BC break to fix a mistake?